etc/scribus.profile

```diff
@@ -48,3 +47,6 @@ tracelog
 # private-bin scribus,gs,gimp*
 private-dev
 private-tmp
+
+# noexec ${HOME}
```

Comment:
does it really need exec from home?

Reply:
Well spotted! Just confirmed that scripting engine works with noexec, I will change it. Thanks.